src/cephadm/cephadm

```diff
@@ -1552,6 +1552,21 @@ def command_pull():
         'image_id': image_id,
         }, indent=4, sort_keys=True))
 
+##################################
+
+def command_inspect_image():
+    # type: () -> int
+    out, err, ret = call_throws([
+        container_path, 'images', '--format', 'json', args.image])
+    j = json.loads(out)
+    if not j:
+        return -errno.ENOENT
+    r = {
+        'image_id': normalize_container_id(j[0].get('id')),
+        'ceph_version': CephContainer(args.image, 'ceph', ['--version']).run(),
+    }
+    print(json.dumps(r, indent=4, sort_keys=True))
+    return 0
```

Comment:
This looks like a copy of 

https://github.com/ceph/ceph/blob/fedcfc0a2d246a153f85f4d96e4f58a80674e82a/src/cephadm/cephadm#L1544-L1553

Can we just call `command_inspect_image()` after pulling the image in `command_pull()`?